src/main/java/com/jcabi/github/mock/MkHooks.java

```diff
@@ -137,7 +139,11 @@ public Hook create(
                 .add("id").set(String.valueOf(number)).up()
                 .add("name").set(name).up()
                 .add("active").set(Boolean.toString(active)).up()
-                .add("events").up()
+                .add("events");
+            for (final Event event : events) {
```

Comment:
@llorllale Same here, we could rewrite this `for` using `events.forEach()`.

Reply:
@paulodamaso see previous comment